lib/widgets.js

```diff
@@ -5,24 +5,11 @@ var reduce = require('reduce');
 var some = require('array.prototype.some');
 var tag = require('./tag');
 
-var dataRegExp = /^data-[a-z]+([-][a-z]+)*$/;
-var ariaRegExp = /^aria-[a-z]+$/;
-var legalAttrs = ['autocomplete', 'autocorrect', 'autofocus', 'autosuggest', 'checked', 'dirname', 'disabled', 'tabindex', 'list', 'max', 'maxlength', 'min', 'novalidate', 'pattern', 'placeholder', 'readonly', 'required', 'size', 'step'];
-var ignoreAttrs = ['id', 'name', 'class', 'classes', 'type', 'value', 'multiple'];
-var getUserAttrs = function (opt) {
```

Comment:
please do `var getUserAttrs = tag.getUserAttrs;` here to avoid breaking if someone mutates the `tag` object later.